onboarding-checklist.md

```diff
@@ -142,8 +142,8 @@ improved and open as PRs on the [template onboarding checklist].
 [template onboarding checklist]: https://github.com/cisagov/development-guide/blob/master/onboarding-checklist.md
 [NCATS Google Drive folder]: https://drive.google.com/drive/folders/0APw76nbCAmzuUk9PVA
 [Dev Team - Events & OOO calendar]: https://calendar.google.com/calendar/embed?src=beta.dhs.gov_pou7e62ob7h1qadu1m8n4o996g%40group.calendar.google.com&ctz=America%2FNew_York
```

Comment:
Given the GWE migration should these be updated now since they will soon be broken?

Reply:
I think any internal links like this should be removed here and just make this a checklist of items that the new Devvie should ask an existing Devvie about.

Reply:
OK, I will make that change.

Reply:
Please see commits 11ec021 and 32fcef3 and let me know what you think.

Reply:
Those both look great to me. 👍

Reply:
@mcdonnnj, can you let me know if we can resolve this conversation or not?  We are also waiting on a re-review from you for this PR.